### ISSUE TITLE
Cannot render quarto documents with the new jupyter-client-8.0.2

### ISSUE THREAD

**Issue by psads-git** [quarto-dev/quarto-cli#4151]
### Bug description

When rendering a quarto document:

```
Starting python3 kernel...ERROR: 

object str can't be used in 'await' expression
```

### Checklist

- [ ] Please include a minimal, fully reproducible example in a single .qmd file? Please provide the whole file rather than the snippet you believe is causing the issue.
- [ ] Please [format your issue](https://quarto.org/bug-reports.html#formatting-make-githubs-markdown-work-for-us) so it is easier for us to read the bug report.
- [ ] Please document the RStudio IDE version you're running (if applicable), by providing the value displayed in the "About RStudio" main menu dialog?
- [ ] Please document the operating system you're running. If on Linux, please provide the specific distribution.

**Comment by cscheid** [comment]
We'll have a 1.2 backport that work with both jupyter_client 7 and 8 by tomorrow. In the meantime, if you need to use 1.2.329, use jupyter_client 8.0.2 (just released).

**Comment by psads-git** [comment]
Thanks, Carlos, for your answer. 

Please, notice that the issue I reported is related with `jupyter-client-8.0.2` -- and _not_ with any previous version of Jupyter.

**Comment by psads-git** [comment]
No problem at all with version `jupyter-client-8.0.1`, Carlos!

**Comment by cscheid** [comment]
@psads-git Yes, I got that totally wrong, it was late last night when I typed this; I apologize. I'll keep the issue open until we verify that our backport does the right thing, but the fix will be out shortly.

**Comment by cderv** [comment]
@psads-git I would try the latest 1.2.335 https://quarto.org/docs/download/

Can you try it ?

**Comment by psads-git** [comment]
Thanks, Christophe: With the new `quarto` version, everything goes fine.

**Comment by cderv** [comment]
Good to know ! I'll close this then.
